libmachine/provision/boot2docker.go

```diff
@@ -90,6 +78,21 @@ func (provisioner *Boot2DockerProvisioner) upgradeIso() error {
 			return err
 		}
 	}
+
+	log.Info("Stopping machine to do the upgrade...")
+
+	if err := provisioner.Driver.Stop(); err != nil {
+		return err
+	}
+
+	if err := mcnutils.WaitFor(drivers.MachineInState(provisioner.Driver, state.Stopped)); err != nil {
+		return err
+	}
+
+	machineName := provisioner.GetDriver().GetMachineName()
+
+	log.Infof("Upgrading machine %s...", machineName)
```

Comment:
I usually like to use `%q` for this purpose or else the name tends to blend in with the rest of the text